pytorch_lightning/callbacks/progress.py

```diff
@@ -282,9 +282,13 @@ def init_train_tqdm(self) -> tqdm:
 
     def init_validation_tqdm(self) -> tqdm:
         """ Override this to customize the tqdm bar for validation. """
+
+        # The main progress bar doesn't exist in trainer.validate(...)
+        has_main_bar = 1 if self.main_progress_bar is not None else 0
+
         bar = tqdm(
             desc='Validating',
-            position=(2 * self.process_position + 1),
+            position=(2 * self.process_position + has_main_bar),
```

Comment:
```suggestion
            position=(2 * self.process_position + int(self.main_progress_bar is not None)),
```

Reply:
This feels a little too convoluted for my taste. I know it does the same, but it's less intuitive to understand at a glance. Let me know if you want me to move forward with the suggestion anyway.

Reply:
You don't have to put it right there, it can be defined above as `has_main_bar = ...`

I think avoiding an if/else outweighs the extra intuitiveness.